app/javascript/packs/mock-device-profiling.tsx

```diff
@@ -62,7 +62,7 @@ const CHAOS_OPTIONS: ChaosOption[] = [
 ];
 
 function MockDeviceProfilingOptions() {
-  const [selectedValue, setSelectedValue] = useState('');
+  const [selectedValue, setSelectedValue] = useState('pass');
```

Comment:
Setting the default selected value as a pass because it was missed in https://github.com/18F/identity-idp/pull/7892 PR for LG-8985 “Make Pass the default option for a mock”.